classes/mail/mailables/UserMail.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @file classes/mail/mailables/UserMail.php
+ *
+ * Copyright (c) 2014-2022 Simon Fraser University
+ * Copyright (c) 2000-2022 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class UserMail
+ * @ingroup mail_mailables
+ *
+ * @brief Send email to a user
+ */
+
+namespace PKP\mail\mailables;
+
+use PKP\mail\Mailable;
+use PKP\mail\traits\Recipient;
+use PKP\mail\traits\Sender;
+
+class UserMail extends Mailable
```

Comment:
This might make interactions with this email more difficult, e.g., from a plugin. But OK, I had the same idea and already used generic `Mailable` class for sending email before.